google/cloud/speech/cloud_speech.yaml

```diff
@@ -12,18 +12,8 @@ documentation:
 
 apis:
 - name: google.cloud.speech.v1beta1.Speech
-# TODO get LRO API call support added for Speech
-#- name: google.longrunning.Operations
+- name: google.longrunning.Operations
```

Comment:
Oh, shoot. I should have checked in on this thread. I just filed https://github.com/googleapis/googleapis/pull/158 showing how LRO could be used with Speech, the same way IAM is used with Pub/Sub.